opentracing/harness/api_check.py

```diff
@@ -19,7 +19,7 @@
 # THE SOFTWARE.
 from __future__ import absolute_import
 import time
-
+from .. import *
```

Comment:
I wanted to do something like `from ... import opentracing` but python won't let me (too many `.`s)... I'll try a few more things.